workflows/createtaxdb.nf

```diff
@@ -250,12 +298,10 @@ workflow CREATETAXDB {
         [],
         []
     )
-    multiqc_report = MULTIQC.out.report.toList()
 
     emit:
     multiqc_report           = MULTIQC.out.report.toList() // channel: /path/to/multiqc_report.html
```

Comment:
You dare not to follow the alignment of our lord and saviour harshil https://nf-co.re/docs/contributing/code_editors_and_styling/harshil_alignment#colon-example-comments?

Reply:
Uhhhhh... Ben made me do it :eyes: